src/classes/BDI_DataImportService.cls

```diff
@@ -748,17 +758,30 @@ global with sharing class BDI_DataImportService {
     * otherwise!)
     * @return void
     */
-    private void disableAllOppRollups() {
-        if (!diSettings.Run_Opportunity_Rollups_while_Processing__c) {
-            List<Trigger_Handler__c> handlers = TDTM_Config_API.getCachedRecords();
-            for (Trigger_Handler__c th : handlers) {
-                if (th.Object__c == 'Opportunity' && (th.Class__c == 'RLLP_OppRollup_TDTM' || th.Class__c == 'CRLP_Rollup_TDTM')) {
-                    th.Active__c = false;
-                }
+    @TestVisible private void disableAllOppRollups() {
+        if (diSettings.Run_Opportunity_Rollups_while_Processing__c) {
+            return;
+        }
+
+        List<Trigger_Handler__c> handlers = TDTM_Config_API.getCachedRecords();
```

Comment:
I think this should just call `TDTM_Config_API.disableAllRollupTriggers()` here this way we have a single point to handle disabling all rollup related triggers. Though, that method can remove the line that disables the Allocation__c trigger for CRLP_Rollup_TDTM since that doesn't exist any more.